Specifications/Language/3_Expressions/ValueLiterals.md

```diff
@@ -20,11 +20,13 @@ Value literals for the [`Int` type](https://github.com/microsoft/qsharp-language
 
 ## BigInt Literals
 
-Value literals for the [`BigInt` type](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) are always postfixed with `L` and can be expressed in decimal or hexadecimal representation. Literals expressed in hexadecimal are prefixed with `0x`. There is no prefix for the commonly used decimal representation.
+Value literals for the [`BigInt` type](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/4_TypeSystem#available-types) are always postfixed with `L` and can be expressed in decimal, binary, octal or hexadecimal representation. Literals expressed in binary are prefixed with '0b', in octal prefixed with '0o', in hexadecimal prefixed with `0x`. There is no prefix for the commonly used decimal representation.
 
 | Representation | Value Literal |
 | --- | --- | 
 | Decimal | `42L` |
+| Binary | '0x101' |
+| Octal | '0o345' |
```

Comment:
For consistency with the Int literals section, can we use the same table numbers in the same order? Also, it looks like the binary and octal examples are missing the `L` suffix.

Reply:
addressed